set.mm

```diff
@@ -788185,6 +789290,1081 @@ Ring homomorphisms (extension)
+        w3a islidl wceq oveq1 oveq1d ralbidv oveq2 rspc2v adantl rspcv 3ad2ant1
+        cgrp rnggrp adantr simpll1 simprl ssel 3ad2ant2 adantld syl3anc grpridd
+        imp rngcl biimpd ex syl5d syld com23 3exp 3impd biimtrid 3imp1 ) BUAPZE
+        DPZHEPZFAPZGEPZUBZFGCQZEPZWCEAUCZEUDUEZMUFZNUFZCQZOUFZBUGUHZQZEPZOERZNE
+        RMARZUJWBWDWGWISZSZMAWPBCDENOLJWPUIZKUKWBWJWKWTXBWBWJWKWTXBSWBWJWKUJZWD
+        WTXAXDWDWTXASXDWDUBZWGWTWIXEWGWTWISXEWGUBZWTWHWOWPQZEPZOERZWIWGWTXISXEW
+        SXIFWMCQZWOWPQZEPZOERMNFGAEWLFULZWRXLOEXMWQXKEXMWNXJWOWPWLFWMCUMUNTUOWM
+        GULZXLXHOEXNXKXGEXNXJWHWOWPWMGFCUPUNTUOUQURXEWGXIWISXEXIWHHWPQZEPZWGWIW
+        DXIXPSXDXHXPOHEWOHULXGXOEWOHWHWPUPTUSURXEWGXPWISXFXPWIXFXOWHEXFAWPBWHHJ
+        XCIXEBVAPZWGXDXQWDWBWJXQWKBVBUTVCVCXFWBWEGAPZWHAPWBWJWKWDWGVDXEWEWFVEXE
+        WGXRXEWFXRWEXDWFXRSZWDWJWBXSWKEAGVFVGVCVHVKABCFGJKVLVIVJTVMVNVOVKVPVNVQ
+        VNVQVRVSVTWA $.
+    $}
+
+    rngridlmcl.u $e |- U = ( LIdeal ` ( oppR ` R ) ) $.
+    $( A right ideal (which is a left ideal over the opposite ring) containing
+       the zero element is closed under right-multiplication by elements of the
+       full non-unital ring.  (Contributed by AV, 19-Feb-2025.) $)
+    rngridlmcl $p |- ( ( ( R e. Rng /\ I e. U /\ .0. e. I )
+                         /\ ( X e. B /\ Y e. I ) ) -> ( Y .x. X ) e. I ) $=
+      ( crng wcel w3a wa co coppr cfv eqid opprmul c0g opprrng id eleq1i biimpi
+      cmulr oppr0 opprbas rnglidlmcl syl3anl eqeltrrid ) BMNZEDNZHENZOFANGENPZP
+      GFCQFGBRSZUGSZQZEABURCUQFGJKUQTZURTZUAUMUQMNUNUNUOBUBSZENZUPUSENBUQUTUCUN
+      UDUOVCHVBEIUEUFAUQURDEFGVBBUQVBUTVBTUHABUQUTJUIVALUJUKUL $.
+  $}
+
+  ${
+    $d R x y z $.
+    rnglidl0.u $e |- U = ( LIdeal ` R ) $.
+    ${
+      $d .0. x y z $.
+      rnglidl0.z $e |- .0. = ( 0g ` R ) $.
+      $( Every non-unital ring contains a zero ideal.  (Contributed by AV,
+         19-Feb-2025.) $)
+      rnglidl0 $p |- ( R e. Rng -> { .0. } e. U ) $=
+        ( vx vy vz wcel cfv cv co wral eqid cvv wceq oveq1d sylibr oveq2 eleq1d
+        crng csn cbs wss c0 wne cmulr cplusg rng0cl snssd c0g fvexi snn0d rngrz
+        wa cgrp rnggrp grpidcl grprid syl2anc2 adantr eqtrd elsn2 ralbidv ralsn
+        a1i bitri ralrimiva islidl syl3anbrc ) AUAIZCUBZAUCJZUDVLUEUFFKZGKZAUGJ
+        ZLZHKZAUHJZLZVLIZHVLMZGVLMZFVMMVLBIVKCVMVMACVMNZEUIUJVKCOCOIVKCAUKEULZV
+        FUMVKWCFVMVKVNVMIZUOZVNCVPLZCVSLZVLIZWCWGWICPWJWGWICCVSLZCWGWHCCVSVMAVP
+        VNCWDVPNZEUNQVKWKCPZWFVKAUPICVMIWMAUQVMACWDEURVMVSACCWDVSNZEUSUTVAVBWIC
+        WEVCRWCWHVRVSLZVLIZHVLMZWJWBWQGCWEVOCPZWAWPHVLWRVTWOVLWRVQWHVRVSVOCVNVP
+        SQTVDVEWPWJHCWEVRCPWOWIVLVRCWHVSSTVEVGRVHFVMVSAVPBVLGHDWDWNWLVIVJ $.
+    $}
+
+    $d B x y z $.
+    rnglidl1.b $e |- B = ( Base ` R ) $.
+    $( The base set of every non-unital ring is an ideal.  For unital rings,
+       such ideals are called "unit ideals", see rglid1 .  (Contributed by AV,
+       19-Feb-2025.) $)
```

Comment:
Do you mean "see `~ lidl1`" ? Or is there a new theorem `~ rglid1` coming in an other PR?

Reply:
~ lidl1 was meant - I'll fix this.